hub/core/compression.py

```diff
@@ -48,6 +83,9 @@ def compress_array(array: np.ndarray, compression: str) -> bytes:
     if compression is None:
         return array.tobytes()
 
+    if compression == "lz4":
+        return _lz4_compress_array(array)
```

Comment:
if statements here are good, we should try to encapsulate as much as possible in this module